modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
+
+Slow Outbound Requests by Flow::
+The average number of outbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Inbound Response Time by Flow::
+The average, minimum, and maximum inbound response time, grouped by flow.
+
+Outbound Response Time by Flow::
+The average, minimum, and maximum outbound response time, grouped by flow.
+
+Inbound Requests by Flow::
+The average number of inbound requests, grouped by flow.
+
+Outbound Requests by Flow::
+The average number of outbound requests, grouped by flow.
+
+Failures by Flow::
+The average number of inbound failed requests, grouped by flow.
+//tbd?
+
+Slow Inbound Requests by Flow::
+The average number of inbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Slow Outbound Requests by Flow::
+The average number of outbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+[[connectors-charts]]
+== Connectors
+
+If you don't see the *Connectors* tab, verify that you have a Titanium subscription to Anypoint Platform and that the app you selected is a hybrid app deployed to a server, server group, or cluster. See xref:monitor-connectors.adoc[Monitor Connectors].
+
+The following charts appear on the *Connectors* dashboard:
+
+Connector Message Volume::
+The sum of outbound messages and outbound requests grouped by connector type in automatic intervals.
+
+Response Time by Connector::
+The average outbound response time grouped by connector type in automatic intervals.
+
+Salesforce Requests::
+The average number of Salesforce connector requests, grouped by operation in automatic intervals.
+
+Salesforce Response Time::
+The average response time of Saleforce connector requests, grouped by connector type in automatic intervals.
+
+Salesforce Failures::
+The average number of Salesforce connector requests failures, grouped by operation in automatic intervals.
+
+Database Requests::
+The average number of Database connector requests, grouped by operation in automatic intervals.
+
+Database Response Time::
+The average response time of Database connector requests, grouped by connector type in automatic intervals.
+
+Database Failures::
+The average number of Database connector requests failures, grouped by operation in automatic intervals.
+
+HTTP Requests::
+The average number of HTTP connector requests, grouped by operation in automatic intervals.
+
+HTTP Response Time::
+The average response time of HTTP connector requests, grouped by connector type in automatic intervals.
+
+HTTP Failures::
+The average number of HTTP connector requests failures, grouped by operation in automatic intervals.
+
+Web Services Requests::
+The average number of Web Services connector requests, grouped by operation in automatic intervals.
+
+Web Services Response Time::
+The average number of Web Services connector request failures, grouped by operation in automatic intervals.
```

Comment:
remove grouped by.  This is not grouped by operation.

Reply:
```suggestion
The average number of Web Services connector request failures, grouped in automatic intervals.
```